README.md

```diff
@@ -288,7 +288,8 @@ We have a status board that checks all IPFS repositories for CI, Readmes, test c
 
 #### IPFS Meetups
 
-* [ipfs/lisbon](https://github.com/ipfs/sf) - The IPFS meetup in Lisbon.
+* [ipfs/lisbon](https://www.meetup.com/ipfs-lisbon-meetup/) - The IPFS meetup in Lisbon.
+* [ipfs/london](https://www.meetup.com/IPFS-London-Meetup/) - The IPFS meetup in London.
```

Comment:
You added a London URL - can you make a more descriptive commit message, for posterity?